server/tables/tables.go

```diff
@@ -585,6 +585,11 @@ type GitRepository struct {
 	// BuildBuddy-initiated invocations within this repo, such as workflows or
 	// remote Bazel.
 	InstanceNameSuffix string `gorm:"not null;default:''"`
+
+	// DefaultNonRootRunner decides whether workflow and remote bazel runs
+	// within this repository should run as a non-root user by default.
+	// TODO(http://go/b/3286): Remove this field after completing migration.
+	DefaultNonRootRunner bool `gorm:"not null;default:0"`
```

Comment:
nit: Would this be more readable inverted as `DefaultRootRunner`

Reply:
slightly more readable but right now I don't want to spend the effort trying to figure out whether we'd also have to update our current creation queries to explicitly set this to true, so will just keep this as is. (This will go away soon anyway)